bindings/python/test/environment/gravitational/test_earth.py

```diff
@@ -0,0 +1,90 @@
+
+def test_earth_constructors ():
+
+    # Test constructor with enums
+    earth_gravitational_model: EarthGravitationalModel = EarthGravitationalModel(EarthGravitationalModel.EarthType.WGS84)
+
+    assert earth_gravitational_model is not None
+    assert isinstance(earth_gravitational_model, EarthGravitationalModel)
+
+    earth_gravitational_model: EarthGravitationalModel = EarthGravitationalModel(EarthGravitationalModel.EarthType.EGM96)
+
+    assert earth_gravitational_model is not None
+    assert isinstance(earth_gravitational_model, EarthGravitationalModel)
+
+    earth_gravitational_model: EarthGravitationalModel = EarthGravitationalModel(EarthGravitationalModel.EarthType.EGM84)
+
+    assert earth_gravitational_model is not None
+    assert isinstance(earth_gravitational_model, EarthGravitationalModel)
+
+    earth_gravitational_model: EarthGravitationalModel = EarthGravitationalModel(EarthGravitationalModel.EarthType.EGM2008)
+
+    assert earth_gravitational_model is not None
+    assert isinstance(earth_gravitational_model, EarthGravitationalModel)
+
+    earth_gravitational_model: EarthGravitationalModel = EarthGravitationalModel(EarthGravitationalModel.EarthType.spherical)
+
+    assert earth_gravitational_model is not None
+    assert isinstance(earth_gravitational_model, EarthGravitationalModel)
+
+    # Test constructor with directory
+    earth_gravitational_model: EarthGravitationalModel = EarthGravitationalModel(EarthGravitationalModel.EarthType.EGM2008, Directory.undefined())
+
+    assert earth_gravitational_model is not None
+    assert isinstance(earth_gravitational_model, EarthGravitationalModel)
+
+    # Test constructor with degree and order
+    earth_gravitational_model: EarthGravitationalModel = EarthGravitationalModel(EarthGravitationalModel.EarthType.EGM2008, Directory.undefined(), 2, 2)
+
+    assert earth_gravitational_model is not None
+    assert isinstance(earth_gravitational_model, EarthGravitationalModel)
+
+################################################################################################################################################################
+
+def test_earth_get_type ():
+
+    earth_gravitational_model = construct_earth()
+
+    assert earth_gravitational_model.get_type() is not None
+    assert earth_gravitational_model.get_type() == EarthGravitationalModel.EarthType.EGM2008
+
+################################################################################################################################################################
+
+def test_earth_get_field_value_at ():
+
+    earth_gravitational_model = construct_earth()
+
+    grav_acceleration = earth_gravitational_model.get_field_value_at(np.array([6400e3, 0.0, 0.0]), Instant.J2000())
+    grav_acceleration_ref = np.array([-9.74722832e+00, -1.29220148e-05, -1.34173896e-05])
```

Comment:
Can you put a comment here as to where you get this value from?
Incase we ever need to update the unit test in the future that will be useful to know.

Reply:
Since I know that the gravity model is currently working, I just printed out the value and put that in the unit test. @vishwaLoft

Reply:
Hmmmmmm fair enough. So it's basically just self-validating the function. I thought for some reason you were checking against values from GMAT or sth.